app/assets/stylesheets/icons.scss

```diff
@@ -192,6 +192,9 @@
 .icon-budget::before {
   content: '\53';
 }
+.icon-budget-investment-image::before {
```

Comment:
Just a tiny bit here ^_^, as scss-lint says "Rule declaration should be followed by an empty line" so just a separation between `.icon-budget::before` and `.icon-budget-investment-image::before` declarations will make @decabeza OCD dissapear when he reads this ^_^